test/function/namespacing-collisions-2/main.js

```diff
@@ -0,0 +1,7 @@
+import { Something } from './Something';
+import { SomethingAgain } from './SomethingAgain';
+import { Material } from './Material';
+import { MaterialAgain } from './MaterialAgain';
+
+var result = [Material(), MaterialAgain(), Something(), SomethingAgain()]
```

Comment:
I don't really use const/let in most of the code I write (I know, go find the pitchforks 😛) and this is just test code, so I figured it was okay. We could apply the eslint rules to the test code if you'd like!